man/docker.1.md

```diff
@@ -223,6 +223,70 @@ inside it)
   Block until a container stops, then print its exit code
   See **docker-wait(1)** for full documentation on the **wait** command.
 
+# CONFIGURATION FILES
```

Comment:
Makes sence to me, but I wonder if I should do this here or in another PR ? :stuck_out_tongue_closed_eyes: